src/common/ssl/SSLConfig.cpp

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "common/ssl/SSLConfig.h"
+
+DEFINE_string(cert_path, "", "Path to cert pem.");
+DEFINE_string(key_path, "", "Path to cert key.");
+DEFINE_string(ca_path, "", "Path to trusted CA file.");
+DEFINE_bool(enable_ssl, false, "Whether enable ssl.");
+DEFINE_bool(enable_graph_ssl, false, "Whether enable ssl of graph server.");
+DEFINE_bool(enable_meta_ssl, false, "Whether enable ssl of meta server.");
```

Comment:
it seems that two flags is enough to cover the cases about inside and outside the nebula cluster. Do we really need to add another meta_ssl for users to make a decision?

Reply:
It's the suggestion from @sherman-the-tank , which to encrypt the sensitive data in meta.

Reply:
I mean whether `enable_ssl` and `enable_graphd_ssl` can already cover the `enable_meta_ssl` scenario, just like below:

```
* enable_graphd_ssl=true & enable_ssl=false

clients <- ssl -> graphd <- no ssl -> metad <- no ssl -> storaged
                     ^                                    ^
                     |______________ no ssl ______________|


* enable_graphd_ssl=false & enable_ssl=true

clients <- no ssl -> graphd <- ssl -> metad <- ssl -> storaged
                       ^                                 ^
                       |______________ ssl ______________|


* enable_graphd_ssl=true & enable_ssl=true

clients <- ssl -> graphd <- ssl -> metad <- ssl -> storaged
                    ^                                 ^
                    |______________ ssl ______________|

* enable_graphd_ssl=false & enable_ssl=false

clients <- no ssl -> graphd <- no ssl -> metad <- no ssl -> storaged
                        ^                                    ^
                        |______________ no ssl ______________|
```

Is `enable_meta_ssl` just to encrypt schema information when communicating with meta? If so, make sense.